vars/main.yml

```diff
@@ -7,6 +7,8 @@ openio_oioswift_definition_file: >
   {{ openio_oioswift_servicename }}/{{ openio_oioswift_servicename }}.conf"
 openio_oioswift_gridinit_dir: "/etc/gridinit.d/{{ openio_oioswift_namespace }}"
 
+default_openio_redis_keys_format: "v2"
```

Comment:
NO
This is not a default.

Reply:
I don t understand, on https://github.com/open-io/ansible-role-openio-oioswift/blob/19.04/defaults/main.yml , redis_keys_format is set by default with v2.

Reply:
Yes, cf QA-17 ticket.